CHANGELOG.md

```diff
@@ -1,277 +1,218 @@
-  ([4e001cd](https://github.com/vivaxy/vscode-conventional-commits/commit/4e001cdfe135bcf8dee23e4e15e81c5e4686608c))
+- :sparkles: support commits in git subdirectories ([4e001cd](https://github.com/vivaxy/vscode-conventional-commits/commit/4e001cdfe135bcf8dee23e4e15e81c5e4686608c))
 
-### [1.1.1](https://github.com/vivaxy/vscode-conventional-commits/compare/v1.1.0...v1.1.1) (2020-03-31)
+## [1.1.1](https://github.com/vivaxy/vscode-conventional-commits/compare/v1.1.0...v1.1.1) (2020-03-31)
 
 ## [1.1.0](https://github.com/vivaxy/vscode-conventional-commits/compare/v1.0.0...v1.1.0) (2020-03-31)
 
 ### Features
 
-- :sparkles: supports remember previous scopes
-  ([9f91e30](https://github.com/vivaxy/vscode-conventional-commits/commit/9f91e303d1bb37a20c3950544175f457ced0f9a1)),
-  closes [#2](https://github.com/vivaxy/vscode-conventional-commits/issues/2)
-- **scope:** :sparkles: limit scope to existing scopes with the ability to add
-  new scopes
-  ([643f689](https://github.com/vivaxy/vscode-conventional-commits/commit/643f68973b75daaca4ce0a01979cc328d61ee583))
-- :sparkles: add step indicators
-  ([6a98690](https://github.com/vivaxy/vscode-conventional-commits/commit/6a9869027110917d1d1545483d47b4f5443a4254))
+- :sparkles: supports remember previous scopes ([9f91e30](https://github.com/vivaxy/vscode-conventional-commits/commit/9f91e303d1bb37a20c3950544175f457ced0f9a1)), closes [#2](https://github.com/vivaxy/vscode-conventional-commits/issues/2)
+- **scope:** :sparkles: limit scope to existing scopes with the ability to add new scopes ([643f689](https://github.com/vivaxy/vscode-conventional-commits/commit/643f68973b75daaca4ce0a01979cc328d61ee583))
+- :sparkles: add step indicators ([6a98690](https://github.com/vivaxy/vscode-conventional-commits/commit/6a9869027110917d1d1545483d47b4f5443a4254))
 
 ### Bug Fixes
 
-- :bug: fix commit message with extra space words
-  ([b4d2802](https://github.com/vivaxy/vscode-conventional-commits/commit/b4d2802d8be374aa7e75127c51fe3f38f4e92279))
+- :bug: fix commit message with extra space words ([b4d2802](https://github.com/vivaxy/vscode-conventional-commits/commit/b4d2802d8be374aa7e75127c51fe3f38f4e92279))
 
 ## [1.0.0](https://github.com/vivaxy/vscode-conventional-commits/compare/v0.0.7...v1.0.0) (2020-03-27)
 
 ### Features
 
-- :sparkles: support disable gitmoji
-  ([d55b16e](https://github.com/vivaxy/vscode-conventional-commits/commit/d55b16ec121c130d48315599be663b02d43c5a5b)),
-  closes [#4](https://github.com/vivaxy/vscode-conventional-commits/issues/4)
+- :sparkles: support disable gitmoji ([d55b16e](https://github.com/vivaxy/vscode-conventional-commits/commit/d55b16ec121c130d48315599be663b02d43c5a5b)), closes [#4](https://github.com/vivaxy/vscode-conventional-commits/issues/4)
 
-### [0.0.7](https://github.com/vivaxy/vscode-conventional-commits/compare/v0.0.6...v0.0.7) (2020-03-27)
+## [0.0.7](https://github.com/vivaxy/vscode-conventional-commits/compare/v0.0.6...v0.0.7) (2020-03-27)
```

Comment:
**This is not a question.**

I've never noticed the `###` issue. Thank you for pointing it out and fixing it.

Reply:
It was also carried by the upstream issue, I think it will auto fix in the latter version. See more discuss in issue(https://github.com/conventional-changelog/standard-version/issues/317).